src/main/webapp/course.html

```diff
@@ -18,54 +26,67 @@
+          <strong class="d-block text-gray-dark">@username</strong>
+          Donec id elit non mi porta gravida at eget metus. Fusce dapibus,
+          tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum
+          massa justo sit amet risus.
         </p>
-        </div>
-        <div class="media text-muted pt-3">
+      </div>
+      <div class="media text-muted pt-3">
         <p class="media-body pb-3 mb-0 small lh-125 border-bottom border-gray">
-            <strong class="d-block text-gray-dark">@username</strong>
-            Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.
+          <strong class="d-block text-gray-dark">@username</strong>
+          Donec id elit non mi porta gravida at eget metus. Fusce dapibus,
+          tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum
+          massa justo sit amet risus.
         </p>
-        </div>
+      </div>
     </div>
 
     <div class="my-3 p-3 bg-white prof-comments">
-        <h1 class="border-bottom border-gray p-5 mb-0 text-center">Professor Comments</h1>
-        <div class="media text-muted pt-3">
+      <h1 class="border-bottom border-gray p-5 mb-0 text-center">
+        Professor Comments
+      </h1>
+      <div class="media text-muted pt-3">
         <p class="media-body pb-3 mb-0 small lh-125 border-bottom border-gray">
-            <strong class="d-block text-gray-dark">@username</strong>
-            Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.
+          <strong class="d-block text-gray-dark">@username</strong>
+          Donec id elit non mi porta gravida at eget metus. Fusce dapibus,
+          tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum
+          massa justo sit amet risus.
         </p>
-        </div>
-        <div class="media text-muted pt-3">
+      </div>
+      <div class="media text-muted pt-3">
         <p class="media-body pb-3 mb-0 small lh-125 border-bottom border-gray">
-            <strong class="d-block text-gray-dark">@username</strong>
-            Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.
-        </div>
+          <strong class="d-block text-gray-dark">@username</strong>
+          Donec id elit non mi porta gravida at eget metus. Fusce dapibus,
+          tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum
+          massa justo sit amet risus.
+        </p>
+      </div>
     </div>
 
     <div id="form-container">
-      <form id="term-form">
+      <form id="termForm">
```

Comment:
Why change this to camelCase but keep the rest dash-case? Typically html properties are dash-case. I know it's weird that different things are different cases. However once you get used to it it lets you quickly identify when the context of the code you're referencing changed.

Reply:
Ohhh that's interesting! Will keep that in mind :)